aws_lambda_powertools/shared/functions.py

```diff
@@ -275,13 +275,11 @@ def abs_lambda_path(relative_path: str = "") -> str:
         If the path is empty, it will return the current working directory.
     """
     # Retrieve the LAMBDA_TASK_ROOT environment variable or default to an empty string
-    current_working_directory = os.environ.get("LAMBDA_TASK_ROOT", "")
+    current_working_directory = os.environ.get("LAMBDA_TASK_ROOT", "") or str(Path.cwd())
 
-    # If LAMBDA_TASK_ROOT is not set, use the current working directory
-    if not current_working_directory:
-        current_working_directory = str(Path.cwd())
+    return str(Path(current_working_directory, relative_path))
 
-    # Combine the current working directory and the relative path to get the absolute path
-    absolute_path = str(Path(current_working_directory, relative_path))
 
-    return absolute_path
+def sanitize_xray_segment_name(name: str) -> str:
+    _common_invalid_name_characters = "?;*()!$~^<>"
+    return "".join(char for char in name if char not in _common_invalid_name_characters)
```

Comment:
Actually, now that I'm thinking on the code, I believe this would be faster:

```python
INVALID_CHARS_PATTERN = r"[?;*()!$~^<>]"
def sanitize_xray_segment_name(name: str) -> str:
    return re.sub(invalid_chars_pattern, "", name)
 ```